src/lib/connectors/crm-v2/documents.js

```diff
@@ -57,7 +62,22 @@ const getDocumentRole = documentRoleId => {
   return serviceRequest.get(getDocumentRolesUrl(documentRoleId));
 };
 
+/**
+ * Fetch a document by documentRef and date, to identify the responsible company
+ *  @param {String} documentRef
+ *  @param {String} date
+ */
+const getDocumentByRefAndDate = async (documentRef, date) => {
+  return serviceRequest.get(getDocumentUrl('search'), {
```

Comment:
If we agree on the above comment then this line changes to: 
return serviceRequest.get(getDocumentsUrl('search'), {